proxy/schemas/responses.py

```
@@ -48,27 +48,34 @@ class Tool(RootModel):
     root: Union[BuiltInTool, CustomTool]
 
 
-class LogProbs(BaseModel):
-    tokens: List[str]
-    token_logprobs: List[float]
-    top_logprobs: List[Optional[Dict[str, float]]]
-    text_offset: List[int]
+class TextFormat(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)